sfcasts/association-field.md

```diff
@@ -1,94 +1,53 @@
-thing also for the asked by field on question, which is another relationship to user.
-So over here in my controller, down near the bottom, cause it's less important. I'll
-say association, field, colon, colon knew how. And then of course, as soon as we do
-that, it will show up kind of boring on this form. We'll get an error over here, cuz
-we need to add a two string method to our user class. So let's pop open user. Um,
-I'll scroll up after my instructor of a function_underscore under score, two string
-returns, a stringing and let's return this arrow. Get full name Back over here.
-Beautiful. You can see it down here on the bottom. It works awesome though. It is
-completely fine. Technically I'm a little tight on the bottom. It's kind of hard for
-me to see this entire thing. So I'm at a little margin bottom to this page, which we
-can do very easily now. Cause we have the assets styles, admin dot C S file. So let's
-see here. Let me do a little bit of digging. There is
-
-A section up here called main content. That's kind of like this entire area this
-time. I'm not gonna like override any CSS variables or anything. I'm just gonna
-target main content, add a little bottom margin. So dot main content margin, bottom
-hundred pixels and that gonna refresh
-
-It gives us a little bit of breathing room down there. I actually needed a force
-refresh, which makes so easier to use this. Now one issue with these fields is, as I
-mentioned, they're really just fancy looking select fields, which means that all the
-users in our database are actually being rendered into the HTML right now watch, if I
-go to view source view, page source in search Ticia you can see that in the HTML, it
-loaded all of these options right onto the page. So if you only have a few users or
-you only have a few topics, no big deal, but in a real application, we're gonna have
-hundreds or thousands or maybe millions of users. We cannot load them onto the entire
-this page. Well break things. Let's fix that next.
\ No newline at end of file
+Let's configure fields for some of our other CRUD controllers. Go to the Questions page. This is just the default list that was added. Go open `QuestionCrudController.php`, un-comment `configureFields`, and then we'll yield some fields here. Hey... that's a pretty good line for my PHP poetry book I've been working on. Let me write that down...
+
+Now where was I? Oh, right! I'm going to say `yield IdField()`, and then `->onlyOnIndex`. Below that, we'll yield a few more fields that are on our Question. 
+
+```
+yield Field::new('name')
+```
+
+I'm using the lazy way, where I just use `Field::new()` and let it guess the field type. That should be good enough unless you need to configure something that's specific to a field type.
+
+I'll copy that... and I'll paste this two more times for `votes` and `createdAt`. For this one, I'll say `->hideOnForm()`. When we head over and refresh... very nice!
```

Comment:
Copy that... and paste this two more times

(the converting from first-person point of view to the second-person pov)